### ISSUE TITLE
Poll interval not honoured

### ISSUE THREAD

**Issue by bmorris591** [containrrr/watchtower#718]
Watchtower self-updated about an hour ago, and I noticed that the poll interval configured via environment properties is not honoured - it schedules the update for 24 hours away

**To Reproduce**
Set `WATCHTOWER_POLL_INTERVAL=300` in the environment

**Expected behavior**
Watchtower outputs that it will schedule the first run in 5 minutes to the logs

**Environment**
 - Platform
   Synology Nas
 - Architecture
   Intel Atom
 - Docker version 
```
Client:
 Version:           18.09.8
 API version:       1.39
 Go version:        go1.11
 Git commit:        bfed4f5
 Built:             Fri Mar 13 06:46:11 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.8
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.11
  Git commit:       3a371f3
  Built:            Fri Mar 13 06:44:35 2020
  OS/Arch:          linux/amd64
  Experimental:     false
```

**Logs from running watchtower with the `--debug` option**
```
time="2020-12-21T18:44:30Z" level=debug msg="Using scope "
time="2020-12-21T18:44:30Z" level=debug msg="Sleeping for a second to ensure the docker api client has been properly initialized."
time="2020-12-21T18:44:31Z" level=debug msg="Retrieving running containers"
time="2020-12-21T18:44:31Z" level=debug msg="There are no additional watchtower containers"
time="2020-12-21T18:44:31Z" level=info msg="Starting Watchtower and scheduling first run: 2020-12-22 18:44:31 +0000 UTC m=+86400.861167784"
```

**Additional context**
Started with

```
docker run -d \
  --name watchtower \
  --restart unless-stopped \
  --privileged \
  -e DOCKER_API_VERSION=1.39 \
  -e WATCHTOWER_NOTIFICATIONS=slack \
  -e WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL="XXXYYY" \
  -e WATCHTOWER_NOTIFICATION_SLACK_CHANNEL=#watchtower \
  -e WATCHTOWER_CLEANUP=true \
  -e WATCHTOWER_POLL_INTERVAL=300 \
  -e WATCHTOWER_DEBUG=true \
  -e WATCHTOWER_TRACE=true \
  -v /volume1/docker/watchtower/config.json:/config.json:ro \
  -v /var/run/docker.sock:/var/run/docker.sock \
  containrrr/watchtower
```

Image SHA

```
"Image": "sha256:0467f75a0a80029813215c32a41821fc97c51196ea5eb77a6711dc4cf695a30a",
```

Slack notifications, for example, carry on working as expected.

**Comment by bmorris591** [comment]
Running with 

```
docker run -d \
  --name watchtower \
  --restart unless-stopped \
  --privileged \
  -e DOCKER_API_VERSION=1.39 \
  -e WATCHTOWER_NOTIFICATIONS=slack \
  -e WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL="XXXYYY" \
  -e WATCHTOWER_NOTIFICATION_SLACK_CHANNEL=#watchtower \
  -e WATCHTOWER_CLEANUP=true \
  -e WATCHTOWER_POLL_INTERVAL=300 \
  -e WATCHTOWER_DEBUG=true \
  -v /volume1/docker/watchtower/config.json:/config.json:ro \
  -v /var/run/docker.sock:/var/run/docker.sock \
  containrrr/watchtower \
  --interval 300
```

i.e. with the poll interval as an argument rather than environment variable results in the right behaviour

```
time="2020-12-21T18:51:55Z" level=debug msg="Using scope "
time="2020-12-21T18:51:55Z" level=debug msg="Sleeping for a second to ensure the docker api client has been properly initialized."
time="2020-12-21T18:51:56Z" level=debug msg="Retrieving running containers"
time="2020-12-21T18:51:56Z" level=debug msg="There are no additional watchtower containers"
time="2020-12-21T18:51:56Z" level=info msg="Starting Watchtower and scheduling first run: 2020-12-21 18:56:56 +0000 UTC m=+300.553127350"
```

**Comment by simonaronsson** [comment]
Ping @piksel. Sounds related to the snakeswap

**Comment by bmorris591** [comment]
Watchtower updated itself again - issue persists. Now at

```
"Image": "sha256:c97723e0dae5329c96bc33e4a3cec3f3eb611469e1ab3e0a263207221878de16"
```

**Comment by simonaronsson** [comment]
This was likely caused by #684 which wasn't meant to be released until `v1.2.0`. I've reverted that commit and released `v1.1.3`, which should reach you in an hour or so. After that, your config should (hopefully) work as before.

Thanks for understanding!

**Comment by piksel** [comment]
This was caused by an overlook by me. The flag is `interval` but the environment variable is `POLL_INTERVAL` (excluding prefix). I didn't notice this during the port.
